test/runner/e2e_delete_rel_test.cpp

```diff
@@ -0,0 +1,177 @@
+#include "test_helper/test_helper.h"
+
+using namespace kuzu::testing;
+
+class DeleteRelTest : public DBTest {
+public:
+    string getInputCSVDir() override {
+        return TestHelper::appendKuzuRootPath("dataset/rel-insertion-tests/");
+    }
+    string getDeleteKnowsRelQuery(string srcTable, string dstTable, int64_t srcID, int64_t dstID) {
+        return StringUtils::string_format(
+            "MATCH (p1:%s)-[e:knows]->(p2:%s) WHERE p1.ID = %d AND p2.ID = %d delete e;",
+            srcTable.c_str(), dstTable.c_str(), srcID, dstID);
+    }
+    string getInsertKnowsRelQuery(string srcTable, string dstTable, int64_t srcID, int64_t dstID) {
+        auto str = StringUtils::string_format("MATCH (p1:%s), (p2:%s) WHERE p1.ID = %d AND p2.ID "
+                                              "= %d create (p1)-[:knows {length: %d}]->(p2);",
+            srcTable.c_str(), dstTable.c_str(), srcID, dstID, srcID);
+        return str;
+    }
+
+    static constexpr uint64_t NUM_RELS_FOR_PERSON_KNOWS_PERSON = 2500;
+    static constexpr uint64_t NUM_RELS_FOR_ANIMAL_KNOWS_PERSON = 51;
+};
+
+TEST_F(DeleteRelTest, deleteRelsFromLargeList) {
```

Comment:
Change d->D in deleteRelsFromLargeList  test name unless we switched to a new style for test names. Here and everywhere,

Reply:
Changed to upper case for test names.